components/automate-ui/src/app/modules/user/user-details/user-details.component.scss

```diff
@@ -117,3 +117,32 @@ section {
     width: 100%;
   }
 }
+
+.telemetry-checkbox-field {
+  margin-top: 25px;
+  width: 500px;
+}
+
+::ng-deep .cdk-overlay-container {
+  position: fixed;
+  z-index: 999999999;
```

Comment:
Is z-index value '999999999' require? can change it to 1000 or 9999?

Reply:
z-index changed to 9999